integration/src/main/resources/testusers/admin.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Andy Anteater",
+  "name": "Admin User",
```

Comment:
I agree with @tcjiang9 that the roles should be obvious and weren't at first glance. Perhaps Andy 'Admin' Anteater?

Reply:
It's fine to rename them if it makes the intent clearer. Actually maybe what we should do here is make it
```suggestion
  "name": "admin",
```

and then in the code check for `user.name.equals("admin")` instead of checking the email address. That would be better I think.

Reply:
I'd prefer to check for the email here, since that's what matters when it comes to the roles